examples/direct_fidelity_estimation.py

```diff
@@ -52,54 +58,108 @@ def compute_characteristic_function(circuit: cirq.Circuit,
     return trace, prob
 
 
+def sample_pauli_states(n_qubits: int, n_trials: int, physical: bool):
+    pauli_gates = [cirq.I, cirq.X, cirq.Y, cirq.Z]
+    if physical:
+        return [[pauli_gates[j]
+                 for j in np.random.choice(4, n_qubits)]
+                for i in range(n_trials)]
+    else:
+        return itertools.product(pauli_gates, repeat=n_qubits)
```

Comment:
"physical" is a bad name. You can do exhaustive or random sampling with or without a simulator.

Reply:
Merge issue here too. I had gotten rid of all 'physical' mentions.